cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4866,6 +4884,14 @@
   </property>
 
   <!-- System pods Configuration  -->
+  <property>
+    <name>system.worker.app.program.twill.controller.start.seconds</name>
```

Comment:
can we open a jira to remove this property and consolidate the logic? We can then consolidate it later on develop while keeping them separate on the release branch